zombies.js

```diff
@@ -178,8 +230,17 @@
  * @name eat
  * @param {Food} itemToEat  The food item to eat.
  */
-
-
+Player.prototype.eat = function(itemToEat) {
+  let index = this.getPack().indexOf(itemToEat);
+  let instance = itemToEat instanceof Food;
+  if (instance && index >= 0) {
+    this.discardItem(itemToEat);
+    this.health += itemToEat.energy; // add energy to health
+    if (this.health >= 100) {
```

Comment:
This should compare against `this.getMaxHealth()` rather than a magic number (100)